llvm/include/llvm/DebugInfo/LogicalView/Core/LVObject.h

```diff
@@ -74,13 +74,15 @@ using LVSymbolGetFunction = bool (LVSymbol::*)() const;
 using LVTypeSetFunction = void (LVType::*)();
 using LVTypeGetFunction = bool (LVType::*)() const;
 
-using LVElements = SmallVector<LVElement *, 8>;
 using LVLines = SmallVector<LVLine *, 8>;
 using LVLocations = SmallVector<LVLocation *, 8>;
 using LVOperations = SmallVector<LVOperation *, 8>;
 using LVScopes = SmallVector<LVScope *, 8>;
 using LVSymbols = SmallVector<LVSymbol *, 8>;
 using LVTypes = SmallVector<LVType *, 8>;
+using LVElements = SmallVector<LVElement *, 8>;
```

Comment:
Can we keep the alphabetical order?
Just move the `LVElements` and `LVElementsView` before the `LVLines.`